core/src/main/scala/org/bykn/bosatsu/CommentStatement.scala

```diff
@@ -25,18 +25,18 @@ object CommentStatement {
    */
   def parser[T](onP: String => P0[T]): Parser.Indy[CommentStatement[T]] =
     Parser.Indy { indent =>
-      val sep = Parser.newline ~ P.string0(indent)
+      val sep = Parser.newline ~ Parser.maybeSpace
```

Comment:
we can weeken this to allow any number of space or tab between comment lines. Why not.